pandas/_typing.py

```diff
@@ -91,6 +91,12 @@
 PandasScalar = Union["Period", "Timestamp", "Timedelta", "Interval"]
 Scalar = Union[PythonScalar, PandasScalar]
 
+# numpy compatible types
+NumpyArrayLike = Union[np.ndarray, Sequence[PythonScalar]]
+NumpyValueArrayLike = Union[PythonScalar, Sequence[PythonScalar]]
+NumpySorter = Union[int, bool, np.ndarray, Sequence[int], Sequence[bool], None]
```

Comment:
im not sure what the policy is on nitpicking these, but could include np.integer

Reply:
good idea.  in next commit